concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransactionType.java

```diff
@@ -24,8 +29,14 @@ public enum TransactionType {
     ENCRYPTED_TRANSFER_WITH_MEMO((byte) 23),
     TRANSFER_WITH_SCHEDULE_AND_MEMO((byte) 24),
     CONFIGURE_BAKER((byte) 25),
-    CONFIGURE_DELEGATION((byte) 26);
+    CONFIGURE_DELEGATION((byte) 26),
 
+    /**
+     * This exists only to deserialize Raw Account Transaction Payload.
+     * Will be removed in future versions.
+     * todo: Deserialize payload using raw byte array.
+     */
+    UNKNOWN_ACCOUNT((byte) 255);
```

Comment:
I think we should rename this to `RAW_PAYLOAD` for now. The `UNKNOWN_ACCOUNT` can be a bit confusing.

Reply:
done.